cernopendata/modules/fixtures/data/docs/delphi-getting-started/delphi-getting-started.md

```diff
@@ -25,27 +25,23 @@ This quick start guide is meant as a guide for the very first steps to get going
 
 The DELPHI stack consists of the following modules:
 
-* DELPHI DST analysis framework, also referred to as dstana. Please take a look at the [Skelana framework](/record/80502) for detailed information.
-* MonteCarlo production, consisting of simulation, reconstruction and short DST production. Note that the DST format depends on the year. During LEP1, mainly short DST was used, while during LEP2 the extendent short DST format was used. The various formats are described [here](/record/80504)
+* DELPHI DST analysis framework, also referred to as `dstana`. Please take a look at the [Skelana framework](/record/80502) for detailed information.
+* MonteCarlo production, includes simulation, reconstruction and short DST creation. Note that the DST format depends on the year. During LEP1, mainly short DST was used, while during LEP2 the extendent short DST format was the default. Details about the different formats can be found [in the dedicated manuals](/record/80504).
 * Event reconstruction from raw data, using the DELPHI event server.
 * The graphical DELPHI Event display, also referred to as [delgra](/record/80503).
 
 ## <a name="before"> Before you start ... </a>
 
-Please read and accept the data access rules.
-
-DELPHI data access rules are available [here](/record/417), or from [the DELPHI web pages](https://delphi-www.web.cern.ch/delphi-www/delsec/finalrules/DELPHI_Data_preservation-8.pdf).
-
-Please read these before accessing the DELPHI software and data.
+Please read and accept the [DELPHI data access rules](/record/417). They can as well be viewd at the [the DELPHI web pages](https://delphi-www.web.cern.ch/delphi-www/delsec/finalrules/DELPHI_Data_preservation-8.pdf).
 
 # <a name="access"> Accessing the software stack</a>
 There are two possible ways to access the software stack.
 
-## <a name="docker"> Docker</a>
-A docker container is available which ships with all the modules installed. Please take a look [here](/docs/delphi-guide-docker).
+## <a name="docker"> `Docker` </a>
```

Comment:
The subsection titles should not be put in backquotes, since the terms would be displayed in red.

I would reserve the backquotes for something people can type, the executable names, the directory names, etc.